haystack/schema.py

```diff
@@ -615,6 +606,8 @@ class MultiLabel:
     contexts: List[str]
     offsets_in_contexts: List[Dict]
     offsets_in_documents: List[Dict]
+    drop_negative_labels: InitVar[bool] = False
+    drop_no_answer: InitVar[bool] = False
 
     def __init__(self, labels: List[Label], drop_negative_labels=False, drop_no_answers=False, **kwargs):
```

Comment:
I know this is not part of the diff, but do you think it makes sense to remove `**kwargs` from this line and replace it with the same parameters defined above? I'd do that for clarity. Right now `**kwargs` seems prone to bugs, if I mistype any attribute I imagine the resulting Pydantic error would be quite hard to comprehend

Reply:
****kwargs** will work on the custom init yet, they just won't be validated.

Reply:
It totally makes sense. It helps to make clear. In my opinion, `**kwargs` should not be used, except where it's the only way to achieve your intent.

But as @tstadel has made the last modification to fix a serialization issue, I think it should be left this way, until an exclusive PR to change this serialization concern.